tests/units/actions/test_action_base.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from abc import ABCMeta
+
+from urnai.actions.action_base import ActionBase
+
+
+class TestActionBase(unittest.TestCase):
+
+    def test_abstract_methods(self):
+        ActionBase.__abstractmethods__ = set()
+
+        class FakeAction(ActionBase):
+            ...
+
+        f = FakeAction()
```

Comment:
The variable must be explicit.
```suggestion
        fake_action = FakeAction()
```